lms/static/js/i18n/zh-cn/djangojs.js

```diff
@@ -4,21 +4,14 @@
 
   var django = globals.django || (globals.django = {});
 
-
```

Comment:
@musmanmalik 
Can you please revisit this piece of code.
This will cause plurals to be fail for Chinease.